### ISSUE TITLE
UX: Improve AI Lab Recipe usability and discovery

### ISSUE THREAD

**Issue by MariaLeonova** [containers/podman-desktop-extension-ai-lab#1728]
### Is your enhancement related to a problem? Please describe

Improve discovery of using the AI app functionality:
![Screenshot 2024-09-16 at 10 49 17](https://github.com/user-attachments/assets/10da807d-1f19-4adb-8e1a-8fd58ff658ee)

Right now it's not easy to find how to open the actual app UI in browser, I've seen multiple users get stuck in a loop trying to do that. The button to open the AI app is hidden in Running -> Actions menu. 
There it's still overshadowed by the Start button, which i this case is redundant, as the recipe is already running. 

The documentation is good, but it's not super easy to find from the UI: https://github.com/containers/podman-desktop-extension-ai-lab?tab=readme-ov-file#usage 

Never in the Summary  -> Try the Chat application does it say: press the icon in actions to try application. 

All the purple underlined words look like links, but some of them are and some are not. Some links ask to confirm you are leaving the UI, some don't. 

![Screenshot 2024-09-16 at 10 55 17](https://github.com/user-attachments/assets/c1bf9100-b324-493a-824b-bacbd2789ef5)

There's pod information, but it's not possible to see full string:
![Screenshot 2024-09-16 at 10 59 48](https://github.com/user-attachments/assets/be97a865-5f6f-4a05-8510-98a849b07fed)

### Describe the solution you'd like

In order to grow AI Lab adoption, we need to evaluate this UI and the user flow, and make improvements suggestions.  

- make it clear how to run start and run the app
- make sure all text makes sense
- make sure all links behave the same way
- find a UI solution to distinguish links from highlighted text

**Comment by axel7083** [comment]
Hi @MariaLeonova !

In the `Summary` tab you have the `README` file of the recipe, for example for the chatbot application, the readme is from the repository [github.com/containers/ai-lab-recipes/blob/main/recipes/natural_language_processing/chatbot/README.md](https://github.com/containers/ai-lab-recipes/blob/main/recipes/natural_language_processing/chatbot/README.md)

As this is unique to each recipes, we cannot change much of the content of it to indicate how to start the application.

## Links issue

Links need to be absolute links, (E.g. https://example.com/hello-world) some links are relative, like `#section-b` and therefore are buggy.

We probably have some works on improving this.

## Contexts

To give some context, to _why do we have a `running` tab ?_  

At the begining of Podman AI Lab we had a single button at the top allowing the user to open the application, this was great but was leading to some scaling problem.

We allow users to start multiple recipes (nobody do that but this is a technical possibility), user could create pods manually, do some stuff, leading to the scenario where we have multiple recipes running.

Therefore the running page was here to allow this scaling, and display the potential multiple pods for this recipe.

### Workflow

While making this change and having this running tab, _we tried_ to redirect the user to this page after creating an application.

Inside the `Start` recipe, we have a `Open details` button once it is started

![image](https://github.com/user-attachments/assets/e5ed7571-f169-4587-a77a-57fd713c0c57)

which redirect to the Running page of the Recipe.

**Comment by MariaLeonova** [comment]
HI @axel7083! Than you for providing background info on some of the UI details - it's definitely helpful to understand the history. 
The concerns I raised are still valid, since we can't go after every user and explain why the things are the way they are. There are a few suggestions that I would still like to make to make the experience smoother and more intuitive. 

For instance, I don't see an issue with having a **Running tab**, if anything, after the user has started the recipe, it is probably more important than the recipe **Summary**.  

One could argue that one of the most he most important things might be **this button**:
![Screenshot 2024-09-16 at 11 56 07](https://github.com/user-attachments/assets/4789a33c-42ce-4899-9b10-1f09f70c8beb)
And, if so, it should gain more visibility.
Could there also be an AI App details page, similar to Service Details? 

Over the next few days I'd like to go over the UI and make some suggestions, which we can then discuss!

**Comment by axel7083** [comment]
> Could there also be an AI App details page, similar to Service Details?

Yes we had some discussion on this matter (somewhere I can't find an associate issue) but currently we do not have it planed, as we do not have much to show in it

**Comment by MariaLeonova** [comment]
@axel7083 @slemeur @jeffmaury I have a question about Recipe usage. 
There's https://podman-desktop.io/docs/ai-lab/start-recipe and https://github.com/containers/podman-desktop-extension-ai-lab?tab=readme-ov-file#usage. Where would be the right place to navigate users to? Is creating a model service necessary to open the AI App and use it in browser? 

Or are these the only pre-requisites? 
```
Prerequisites

- Podman AI Lab installed.
- Model downloaded.
```

**Comment by jeffmaury** [comment]
Even model download is not mandatory

**Comment by slemeur** [comment]
I agree that opening a running instance of the recipe has been getting more complexe. But it is consistent with how Podman Desktop handles it for opening an application running in a container.

Also, when you have already an instance of the recipe running, when you are on this page - there is no way to see it. 
<img width="1303" alt="Screenshot 2024-09-18 at 16 40 35" src="https://github.com/user-attachments/assets/36bbb023-478a-4214-b78a-dcc8fa3d5a26">


For your questions:

> There's https://podman-desktop.io/docs/ai-lab/start-recipe and https://github.com/containers/podman-desktop-extension-ai-lab?tab=readme-ov-file#usage. Where would be the right place to navigate users to?

one is the readme of the repository, the other is the documentation. I'd point users to the documentation.

> And also is creating a model service necessary to open the AI App and use it in browser?
A model service is needed to run the application yes. But if you already have one started for the model you select when starting the recipe - then it will use that model service.

> Prerequisites
> - Podman AI Lab installed.
> - Model downloaded.

When starting a recipe, if you choose a model that is not yet download, the tool will take care of pulling the model. So it's not a pre-requisite.

**Comment by axel7083** [comment]
If we really want to emphasise with the fact there are something in the Running tab we could add an indicator

![image](https://github.com/user-attachments/assets/a8a48c4e-5be2-46a2-b3bf-a88ca31566ee)

`Running (1)`.
